packages/ckeditor5-html-support/src/datafilter.js

```diff
@@ -106,8 +106,18 @@ export default class DataFilter extends Plugin {
 		*/
 		this._dataInitialized = false;
 
+		/**
+		 * Cached map of coupled attributes. The map is keyed by the feature attribute name
+		 * and coupled GHS attribute names are stored in the value array.
```

Comment:
```suggestion
		 * Cached map of coupled attributes. Keys are the feature attributes names
		 * and values are arrays with coupled GHS attributes names.
```

Reply:
I am not sure if after my changes this is correct, but I think that simply writing "keys are, values are" is simpler to understand.